fs/accounting.go

```diff
@@ -129,23 +131,37 @@ func (s *StatsInfo) String() string {
 	}
 	dtRounded := dt - (dt % (time.Second / 10))
 	buf := &bytes.Buffer{}
+	
+	transferVolume, transferSpeed := s.bytes, speed
+	transferVolumeUnit, transferSpeedUnit := "Bytes", "Bytes/s"
+
+	if *statsBps {
+		transferVolume, transferSpeed = s.bytes*8, speed*8
+	  	transferVolumeUnit, transferSpeedUnit = "Bits", "Bits/s"
+	}
+	
 	fmt.Fprintf(buf, `
-Transferred:   %10s (%s)
-Errors:        %10d
-Checks:        %10d
-Transferred:   %10d
-Elapsed time:  %10v
+%-20s: %-10s (%-s)
+%-20s: %-10d
+%-20s: %-10d
+%-20s: %-10d
+%-20s: %-10v
 `,
-		SizeSuffix(s.bytes).Unit("Bytes"), SizeSuffix(speed).Unit("Bytes/s"),
+		"Transferred",
+		SizeSuffix(transferVolume).Unit(transferVolumeUnit), SizeSuffix(transferSpeed).Unit(transferSpeedUnit),
```

Comment:
Also I don't think we ever want the total transferred in bits, only the transfer rates

    Transferred         : 60.000 MBits (2.683 MBits/s)

What do you think?